test/models/concerns/user_multifactor_methods_test.rb

```diff
@@ -7,6 +7,98 @@ class UserMultifactorMethodsTest < ActiveSupport::TestCase
     @user = create(:user)
   end
 
+  context "validation" do
+    context "#mfa_level_for_enabled_devices" do
+      context "when mfa_level is disabled" do
+        should "be valid if there no mfa devices" do
+          assert_predicate @user, :valid?
```

Comment:
Presently, the `setup` does _not_ include configuring TOTP or WebAuthn, so "no mfa devices" is being inferred by the absence of its set up. 

To make this test more explicit, we could add an assertion to make it clear that there are indeed no mfa devices set up. 

Something like this should do the job:
```ruby
assert_predicate @user, :no_mfa_devices?
```